File: verify/count.py
def verify(args):
    j = 0
    i = 1
    res = ""
    fmtloop = "i = {}\n"
    fmtend = "i ended at {}\n\n"
    while (j != 10):
        res += fmtloop.format(i)
        res += "j = {}\n".format(j)
        j = i
        i += 1
    res += "j ended at {}\n".format(j)
    res += fmtend.format(i)
    for i in range(10):
        res += "i = {}\n".format(i)
    res += "i at 10\n\n"
    for i in range(15, -1, -1):
        res += fmtloop.format(i)
    res += fmtend.format(-1)
    i = -25;
    while (i != -3):
        i += 1
        res += fmtloop.format(i)
    i += 1
    res += fmtend.format(i)
    return (0, res, "")


data = [
    [],
]

File: tests/flow/count.c
int main() {
    int i = 0, j;
    char* fmtloop = "i = %d\n";
    char* fmtend = "i ended at %d\n\n";
    while ((j = i++) != 10) {
        printf(fmtloop, i);
        printf("j = %d\n", j);
    }
    printf("j ended at %d\n", j);
    printf(fmtend, i);
    for (i = 0; i < 10; i++) {
        printf("i = %d\n", i);
    }
    printf("i at %d\n\n", i);
    for (i = 15; i != -1; i--) {
        printf(fmtloop, i);
    }
    printf(fmtend, i);
    i = -25;
    while ((i++) != -3) {
        printf(fmtloop, i);
    }
    printf(fmtend, i);
}
